getsolus/packages
# beeref: Update to 0.31

**Summary**

- Full changelog can be found [here](https://raw.githubusercontent.com/rbreu/beeref/v0.3.2/CHANGELOG.rst)
- Added appstream metadata (Part of getsolus/packages#1389)
- Resolves getsolus/packages#995


**Test Plan**

- Create a picture collage
- Verify metadata with `appstream-builder --packages-dir=. --include-failed -v`

**Checklist**
